android/src/main/java/com/google/samples/apps/ibeaconapp/LifecycleHandler.java

```diff
@@ -0,0 +1,86 @@
+package com.google.samples.apps.ibeaconapp;
+
+import android.app.Activity;
+import android.app.Application;
+import android.os.Bundle;
+import com.google.samples.apps.ibeaconapp.lightbluebean.IBeaconManager;
+
```

Comment:
Javadoc (https://en.wikipedia.org/wiki/Javadoc) at least for newly created classes is highly appreciated .